README.md

```diff
@@ -188,7 +188,7 @@ Where `myApp` is the CMake target in your project.
 Using `rbuild`, you can install the dependencies for development with:
 
 ```bash
-rbuild develop
+rbuild develop -DGPU_TARGETS=$(/opt/rocm/bin/rocminfo | grep -o -m1 'gfx.*')
```

Comment:
Need to test if setting GPU_TARGETS is required still or not. I recall now that  there was a  plan to include a change in compiler so that it wouldn't be required to set this.

Reply:
You can find some pointers perhaps through earlier PR https://github.com/ROCm/AMDMIGraphX/pull/2018

Reply:
I tried with latest develop branch (88048b88b607fac5c70a70aae8ced00e9ad21708), and I still see the error:
```
CMake Error at src/targets/gpu/CMakeLists.txt:31 (message):
  HIP package is broken and has no GPU_TARGETS.  Please pass GPU_TARGETS to
  cmake.

  Use -DGPU_TARGETS=$(/opt/rocm/bin/rocminfo | grep -o -m1 'gfx.*') to build
  for your GPU.


-- Configuring incomplete, errors occurred!
See also "/code/AMDMIGraphX/AMDMIGraphX/build/CMakeFiles/CMakeOutput.log".
See also "/code/AMDMIGraphX/AMDMIGraphX/build/CMakeFiles/CMakeError.log".
```
Doing `rbuild develop -DGPU_TARGETS=$(/opt/rocm/bin/rocminfo | grep -o -m1 'gfx.*')` gets rid of the error.